src/test/scala/com/rxthings/di/ExampleOfTestingSpec.scala

```diff
@@ -19,6 +19,7 @@ class ExampleOfTestingSpec extends InjectSpec with Matchers {
 }
 
 class InjectedActor extends NopActor {
+  import context._
```

Comment:
Well, yes, I see your point. On the other hand, with this approach, for `inject` to work you need to have an implicit `ActorSystem` in the scope, which is in line with other Akka projects, e.g. Akka Streams and Akka Http. 
And, as you can see, only 2 of all the tests were modified to support this. I'd argue it's a minimal invasive change, and it aligns akka-inject API with rest of Akka ecosystem.
So in summary, inside actors this will be resolved by `import context._` (or `import context.system` if you don't do wildcard imports).